### ISSUE TITLE
Height for the single element in row

### ISSUE THREAD

**Issue by karki-dennis** [liabru/jquery-match-height#105]
I have a One item in a row, and it is not giving height to that item. Can we give height to that item? It seems it gives height to the elements if there are only two or more items in the row.

Thanks

**Comment by liabru** [comment]
This is intentional when you're using the `byRow` option, can you should try setting it to `false`?

**Comment by mirafreshface** [comment]
Hi! I have the same issue, I need the height to be applied even when there is only one element in a set. I am using vertical centering solution inside the element that does not work if the height is not explicitly set.

I understand why you made it like this, but an option to turn this behavior on/off would be great.

Thanks for the plugin, it's great!

**Comment by mirafreshface** [comment]
A temporary solution that worked for me was changing the line 127:

from:

`if (this.length <= 1 && !opts.target) {`

to this:

`if (this.length <= 0 && !opts.target) {`

**Comment by liabru** [comment]
This feels like something you should just write your own additional code for, since the reason you need an explicit height is not related to matching heights of multiple elements?

**Comment by mirafreshface** [comment]
Yeah, it's highly debatable, I can see the pros and cons. Honestly, I don't know if it's worth adding for everyone since I am biased by my particular use case.

Are there any downsides if you would simply change `1`to `0` in your code for everyone? I don't see the big picture but you might know better.

I understand there is no point in matching heights when there is only one element in a set but let's say you did that, would it create any problems for people that I can't foresee?

**Comment by smerriman** [comment]
+1 for this. I have a set of elements which wrap onto multiple lines based on the screen size. Each contains an image div which has matchHeight called on it, and the position: relative; top:50%; transform:translateY(-50%) trick to center the images vertically.

When items wrap so that there is a single item on a row, that item doesn't have a height set, which breaks the CSS trick.
